Create ui-c8y-1023-22-10-fix-preselection-of-nested-properties-in-asset-property-selector-graft-release.md (#4162)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-22-10-fix-preselection-of-nested-properties-in-asset-property-selector-graft-release.md

---
date: ""

> title: Fix pre-selection of nested properties in asset property selector

Reviewer: ```suggestion
title: Nested properties displayed as pre-selected in the asset property selector
```

> title: Fix preselection of nested properties in asset property selector [GRAFT][release/cd] (#10967)

Reviewer 2: ```suggestion
title: Fix pre-selection of nested properties in asset property selector
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65890
version: 1023.22.10
---

> With asset property selector, you can provide list of preselected properties as an input for selector. Before the fix, nested properties were not preselected on the list. It has been fixed and nested properties (children of complex properties) can be set as preselected.

Reviewer 2: ```suggestion
The asset property selector allows you to pre-select properties that should be highlighted when the selector opens. Previously, nested properties (properties that are children of complex properties) were not being pre-selected correctly, even when they were included in the pre-selection list. This has been fixed, and nested properties now display as pre-selected in the asset property selector, making it easier to work with hierarchical property structures.
```

> The asset property selector allows you to pre-select properties that should be highlighted when the selector opens. Previously, nested properties (properties that are children of complex properties) were not being pre-selected correctly, even when they were included in the pre-selection list. This has been fixed, and nested properties now display as pre-selected in the asset property selector, making it easier to work with hierarchical property structures.

Reviewer: ```suggestion
The asset property selector allows you to pre-select properties that will be highlighted when the selector opens. Previously, nested properties (properties that are children of complex properties) were not pre-selected correctly, even when they were included in the pre-selection list. This issue has been fixed, and nested properties now display as pre-selected in the asset property selector, making it easier to work with hierarchical property structures.
```
